parsers/CENACE.py

```diff
@@ -294,8 +302,9 @@ def fetch_consumption(
     if demand_td is None:
         raise ParserException("CENACE.py", "Could not find demand cell", zone_key)
     demand = float(demand_td.text.replace(",", ""))
```

Comment:
This is a potential source of future confusion because we're overwriting the timezone value in the yaml file. I feel that checking timezone in ZONES_CONFIG and RETIRED_ZONES_CONFIG and setting after is worth it, even if it is more verbose.

Reply:
Would you consider this is something we need to fix before merging this or is this more of follow up task?

Reply:
I would do it before merging. If you're swamped I can also make the change.

Reply:
I'll take a look at it later today but either way I'll probably merge this tomorrow morning so it's not blocking any further tasks longer than that.

If that's the case I'll follow up and do it in a new PR.